aviary/models/large_single_aisle_1/large_single_aisle_1_GwGm.csv

```diff
@@ -14,7 +14,8 @@ aircraft:design:drag_increment,0.00175,unitless
 aircraft:design:equipment_mass_coefficients,928,0.0736,0.112,0.14,1959,1.65,551,11192,5,3,50,7.6,12,unitless
 aircraft:design:max_structural_speed,402.5,mi/h
 aircraft:design:part25_structural_category,3,unitless
-aircraft:design:reserves,4998,unitless
+aircraft:design:reserve_fuel_additional,4998,lbm
+aircraft:design:reserve_fuel_fraction,0,unitless
```

Comment:
If it is unspecified, it should default to 0, I'll double check. I believe this is just here because it was generated by the fortran to aviary converter

Reply:
confirmed that removing it will default it to 0 and doesn't change the results. It is only needs to be here for testing the fortran to aviary converter.